uber/site_sections/preregistration.py

```diff
@@ -381,7 +381,7 @@ def unset_group_member(self, session, id):
         except:
             log.error('unable to send group unset email', exc_info=True)
 
-        session.assign_badges(attendee.group, attendee.group.badges + 1)
+        session.assign_badges(attendee.group, attendee.group.badges + 1, registered=attendee.registered)
```

Comment:
This is a related fix.  When we unassign a badge, we need to make sure the unassigned badge we create has the same ``registered`` date as the badge being unassigned, because the date on which each badge was created affects the group price for paid groups.